cdc/puller/sorter/merger.go

```diff
@@ -477,95 +508,7 @@ func printError(err error) error {
 		!strings.Contains(err.Error(), "context deadline exceeded") &&
 		cerrors.ErrAsyncIOCancelled.NotEqual(errors.Cause(err)) {
 
-		log.Warn("Unified Sorter: Error detected", zap.Error(err))
+		log.Warn("Unified Sorter: Error detected", zap.Error(err), zap.Stack("stack"))
```

Comment:
What's the difference between the error stack and calling `zap.Stack` directly

Reply:
This error might not originate from this call stack.